js/form-validation.js

```diff
@@ -0,0 +1,23 @@
+// Cliente side validation
+
+// Get elements for the form, need the actual form, the email
+// and the div id to show the error.
+const form = document.getElementById('form');
+const email = document.getElementById('email');
+const errorElement = document.getElementById('error');
+
+form.addEventListener('submit', (event) => {
+  const messages = [];
```

Comment:
[OPTIONAL] Not sure I see the point of using an array here 🤔 considering that only one error is always showing but as they say "If it ain't broke, don't fix it" 💯